### DISCUSSION TITLE
Re-Implement Deletion of Tracks within A List, or An On/Off Toggle for Individual Points/Tracks

### DISCUSSION THREAD

**Discussion by Scrapporoid** [organicmaps/organicmaps#2809] [Ideas]
When in a bookmark list in Organic Maps, it's possible to delete a point by doing a long press and then selecting the "delete" option.  About seven* years ago, Maps.me also used to allow deletion of tracks within a bookmark list, but the feature disappeared at some point, and the feature doesn't exist in Organic Maps.  My suggestion is to re-implement the feature in Organic Maps, or as an alternate an on/off toggle for individual points/tracks within a bookmark list.

Why would one want such a feature?  

- It's handy to de-clutter when one has multiple tracks, some of which you've already traveled or know you will not be traveling during your outing.
- It allows the ability to have day/night colored points and tracks which you can select depending on whether Night Mode is on or off.
- It allows you to create a "sequence" of tracks where you can delete what you've already done so you don't erroneously or needlessly cover the same track twice.  Two examples where I've done this, featuring differently-colored tracks and ordered so the current one was "on top":  1) A "pub crawl" and 2) a multi-route driving tour of a city.

Thank you for your consideration of this.

Regards,
Shawn

* I say seven, as I looked at the date when I created a number of files in a manner to utilize this ability, and they were all in 2015.

**Comment by biodranik** [comment]
iOS version can already delete a track. It should be done for Android too. Can you please create an issue for it?

**Reply by Scrapporoid** [reply]
I somehow missed this way back when you replied -- apologies!  Now I see you opened [3672](https://github.com/organicmaps/organicmaps/issues/3672) -- thanks!
